robottelo/datafactory.py

```diff
@@ -161,3 +203,19 @@ def valid_names_list():
         u' {0}'.format(gen_string('utf8')),
         u'{0} '.format(gen_string('utf8')),
     ]
+
+
+@datacheck
+def valid_users_list():
```

Comment:
Also for further improvement, we may use valid_usernames, valid_ids, etc... If, for definition, data factories return list of data points than there is not need to add list to the name.